src/components/Categories/Categories.js

```diff
@@ -1,9 +1,14 @@
 import React from 'react';
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../../redux/categories/categories';
 
 function Categories() {
+  const dispatch = useDispatch();
   return (
     <div>
-      <button type="submit">Check status</button>
+      <button type="submit" onClick={() => { dispatch(checkStatus()); }}>
+        Check status
+      </button>
```

Comment:
- [ ] Clicking on the `check status` button suppose to show `under construction` the text data that you suppose to get when `checkStatus()` function is dispatched. So please fix that.